coopdigital/coop-frontend
# Storybook beta with postcss8

## Storybook and react additions

What we've done:
- Added react and react-dom
- Added Storybook react, for react component development
- Updated `babel.config.js` for react builds
- Updated the `component-template` package to help start new react components.
- Added a `component-blockquote`, includes a react component, styles and jest tests.


To test:

Start the storybook app
```
npm ci
npm run storybook
```
confirm working with stories at `localhost:6006`
